projects/index.html

```diff
@@ -1,11 +1,11 @@
 ---
-title: Projects
-excerpt: "This is where we keep a collection of some of the projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
+title: Project Showcase
+excerpt: "This is where we keep a collection of some of the most interesting projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
```

Comment:
Another idea: I like how on [lego mindstorms page](http://www.lego.com/en-us/mindstorms/build-a-robot) they have several 'official' robots that may be easily reproduced by a newcomer, and then they have a gallery of community projects. We could even provide programs for those stock robots (and build instructions are already available on lego.com).